Enable external flash in nRF91 DK overlays using it (#11252)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

  * The application now uses the function :c:func:`nrf_cloud_location_request_msg_json_encode` to create an nRF Cloud location request message.
  * The application now uses defines from the :ref:`lib_nrf_cloud` library for string values related to nRF Cloud.
  * Instead of sending a battery voltage, the PMIC's fuel gauge function is used to get a battery percentage. For nRF Cloud, the data ID "VOLTAGE" has been replaced with "BATTERY". For the other cloud backends, the name stays the same, but the range changes to 0-100.

>   * Enabled external flash in the nRF9160 DK device tree overlays for v0.14.0 and newer, as it is now disabled by default in the device tree.

Reviewer: I'm not sure about «the device tree». Is that correct terminology? Here is a suggestion, but I'm not sure it's that good either. Any other alternatives?

```suggestion
  * Enabled external flash in the nRF9160 DK device tree overlays for v0.14.0 and newer, as it is now disabled in the Zephyr board definition.
```

>   * Enabled external flash in the nRF9160 DK device tree overlays for v0.14.0 and newer, as it is now disabled in the Zephyr board definition.

Reviewer 2: ```suggestion
  * Enabled external flash in the nRF9160 DK devicetree overlays for v0.14.0 or later versions, as it is now disabled in the Zephyr board definition.
```
The same applies for other entries.
